Resources/doc/reference/security.rst

```diff
@@ -56,6 +56,10 @@ The related download route name is ``sonata_media_download``.
 
     <a href="{{ path('sonata_media_download', {'id': media|sonata_urlsafeid }) }}">Download file</a>
 
+.. note::
+
+    You must have an admin registered for the media object in order to work with this route.
```

Comment:
So, what route might I use to get a secured path?
In my use case, I have this config:
```yaml
# config.yml
sonata_media:
    contexts:
        content:
            providers:
                - sonata.media.provider.file
            download:
                strategy: sonata.media.security.connected_strategy
                mode: http
```
So in view I added this route, but it complains about the admin, so I had to configure one for my Media object. Notice that my strategy isn't ```sonata.media.security.superadmin_strategy```.